paml/__init__.py

```diff
@@ -79,6 +79,16 @@ def protocol_primitive_step(self, primitive: Primitive, **input_pin_map):
     return pe
 Protocol.primitive_step = protocol_primitive_step  # Add to class via monkey patch
 
+def protocol_template():
+    """
+    Create a template instantiation of a protocol
```

Comment:
I don't understand what this function is or why it in this package. Is this part of the demo material?

Reply:
Yes, I created a Jupyter widget based UI that has buttons to add templates for these types of things.

Reply:
Two requests, then:
1. Can you please add that use case information to the documentation?
2. Can these templates move to a Jupyter extension file rather than going into the main files?

Reply:
1. Yes
2. I was thinking about that.  I think they are useful to more than just Jupyter.  Maybe a "ui" extension file?